src/Microsoft.ML.ImageAnalytics/ImageLoader.cs

```diff
@@ -88,9 +88,13 @@ internal sealed class Options : TransformInputBase
         /// <param name="env">The host environment.</param>
         /// <param name="imageFolder">Folder where to look for images.</param>
         /// <param name="columns">Names of input and output columns.</param>
-        internal ImageLoadingTransformer(IHostEnvironment env, string imageFolder = null, params (string outputColumnName, string inputColumnName)[] columns)
+        internal ImageLoadingTransformer(IHostEnvironment env, string imageFolder, params (string outputColumnName, string inputColumnName)[] columns)
             : base(Contracts.CheckRef(env, nameof(env)).Register(nameof(ImageLoadingTransformer)), columns)
         {
+            if (String.IsNullOrWhiteSpace(imageFolder) || !File.Exists(imageFolder))
+            {
+                throw new ArgumentException("The provided ImageFolder path {0} is invalid.", imageFolder);
+            }
```

Comment:
I wonder: why did you write all of this in here but not in the other ImageLoadingTransformer overload were you actually used `Path.GetFullPath(imageFolder);`? Is it not possible to use GetFullPath in here as well, in order to avoid having to write this 'if' statement?

Reply:
Good point, after I saw I can use `Path.GetFullPath(imageFolder);` I forgot to update it for this case, thank you!